### PR TITLE
Deepfrying nonfoods is no longer a qdel

### PR DESCRIPTION
**Pull request by zeskorion**
## About The Pull Request
sticking something in the deep fryer or coating it with cooking oil now does nothing more than batter the item. it does not render the item in any way inoperable or irretrievable. Cleaning the item, eating it, etc, removes the oil.

## Why It's Good For The Game
deep frying via oil spray became a meta after some asshole made a video on it. it was already shifty to have a qdel machine right fucking there in the kitchen, that has no checks on item resistance whatsoever.

note that this does not change deep frying behaviour for foods

## Changelog
:cl:
balance: fried items can now be retrieved through eating or cleaning
/:cl:

### CODE REVIEW

File: code/modules/food_and_drinks/food/snacks_bread.dm [hunk 1]
```diff
@@ -237,20 +237,21 @@
...
 	else
 		fried.forceMove(src)
-	
-/obj/item/reagent_containers/food/snacks/deepfryholder/Destroy()
-	if(contents)
-		QDEL_LIST(contents)
-	. = ..()
+
+/obj/item/reagent_containers/food/snacks/deepfryholder/proc/clean_batter()
+	qdel(src)
```
Comment (PowerfulBacon) [inline code/modules/food_and_drinks/food/snacks_bread.dm:249]: This brings the items inside back?

**Comment by Moccha-Bee** [comment]
you are unironically retarded, and this pr is utter shit

**Comment by ghost** [comment]
> you are unironically retarded, and this pr is utter shit

mad?

File: code/modules/food_and_drinks/food/snacks_bread.dm (hunk 1 above)
Comment (zeskorion) [reply code/modules/food_and_drinks/food/snacks_bread.dm:249]: Yes

**Comment by ToasterBan** [comment]
just make it so deepfried items still work but can be eaten

**Comment by ghost** [comment]
speedmerge when?

**Comment by ghost** [comment]
people do this every round now and it's getting fucking horrible, this is needed.

**Comment by DungeonPenguin** [comment]
let me just deepfry this thread because its horrible
zesk did someone deepfry your id as captain or some shit

**Comment by ghost** [comment]
> let me just deepfry this thread because its horrible
> zesk did someone deepfry your id as captain or some shit

Tell me one good reason why what is undoubtedly an un-intended feature which is used to mass-destroy the items of others shouldn't be fixed, go on, especially when it's done constantly. Shouldn't have abused this so frequently and every round if you didn't want this removed, you brought this onto yourself.

**Comment by ghost** [comment]
Then just make it so you can’t spray it on genius

**Comment by bloons3** [comment]
> people do this every round now and it's getting fucking horrible, this is needed.

It's specifically logged, and doing it for no reason counts as griefing.

**Comment by DungeonPenguin** [comment]
honk

**Comment by ghost** [comment]
> > people do this every round now and it's getting fucking horrible, this is needed.
> 
> It's specifically logged, and doing it for no reason counts as griefing.

People only use it for griefing, might aswell mechanically restrict it at this point.

**Review by PowerfulBacon — APPROVED** [review]

**Comment by ike709** [comment]
Why are we voting on something with objectively better performance and gameplay?

**Comment by Moccha-Bee** [comment]
> Why are we voting on something with objectively better performance and gameplay?

deep frying things is funny and a core part of chef gameplay no reason it should be removed.

**Comment by ghost** [comment]
> > Why are we voting on something with objectively better performance and gameplay?
> 
> deep frying things is funny and a core part of chef gameplay no reason it should be removed.

Are you fucking retarded? This makes it so deepfrying DOESNT COMPLETELY DESTROY THE DEEPFRIED THING IT DOESNT PREVENT DEEPFRIENG

**Comment by Moccha-Bee** [comment]
> > > Why are we voting on something with objectively better performance and gameplay?
> > 
> > 
> > deep frying things is funny and a core part of chef gameplay no reason it should be removed.
> 
> Are you fucking retarded? This makes it so deepfrying DOESNT COMPLETELY DESTROY THE DEEPFRIED THING IT DOESNT PREVENT DEEPFRIENG

>making items no longer be eaten and deleted if eaten

same thing as removing deep frying everyone loves deep frying items to mess with people or get rid of evidence this is some ided type of pr

**Comment by ghost** [comment]
> > > > Why are we voting on something with objectively better performance and gameplay?
> > > 
> > > 
> > > deep frying things is funny and a core part of chef gameplay no reason it should be removed.
> > 
> > 
> > Are you fucking retarded? This makes it so deepfrying DOESNT COMPLETELY DESTROY THE DEEPFRIED THING IT DOESNT PREVENT DEEPFRIENG
> 
> > making items no longer be eaten and deleted if eaten
> 
> same thing as removing deep frying everyone loves deep frying items to mess with people or get rid of evidence this is some ided type of pr

A machine that completely destroys items that stands in a room connected to the public hallway with absolutley nothing inbetween but a single row of tables, you are fucking delusional, by your logic we might aswell place the cremator into departures.

**Comment by Moccha-Bee** [comment]
> > > > > Why are we voting on something with objectively better performance and gameplay?
> > > > 
> > > > 
> > > > deep frying things is funny and a core part of chef gameplay no reason it should be removed.
> > > 
> > > 
> > > Are you fucking retarded? This makes it so deepfrying DOESNT COMPLETELY DESTROY THE DEEPFRIED THING IT DOESNT PREVENT DEEPFRIENG
> > 
> > 
> > > making items no longer be eaten and deleted if eaten
> > 
> > 
> > same thing as removing deep frying everyone loves deep frying items to mess with people or get rid of evidence this is some ided type of pr
> 
> A machine that completely destroys items that stands in a room connected to the public hallway with absolutley nothing inbetween but a single row of tables, you are fucking delusional, by your logic we might aswell place the cremator into departures.

by this response alone you show that you dont play the game, 1 its really easy ro get access to item deleltion, 2 the chef will always kick your skull in if you break into the kitchen ive always seen it happe  specially when i try to fry shit 3 getting into the crematarium is just a welder and wrench away lol, or just ask the ai it always opens the door anyway

**Comment by ghost** [comment]
> > > > > > Why are we voting on something with objectively better performance and gameplay?
> > > > > 
> > > > > 
> > > > > deep frying things is funny and a core part of chef gameplay no reason it should be removed.
> > > > 
> > > > 
> > > > Are you fucking retarded? This makes it so deepfrying DOESNT COMPLETELY DESTROY THE DEEPFRIED THING IT DOESNT PREVENT DEEPFRIENG
> > > 
> > > 
> > > > making items no longer be eaten and deleted if eaten
> > > 
> > > 
> > > same thing as removing deep frying everyone loves deep frying items to mess with people or get rid of evidence this is some ided type of pr
> > 
> > 
> > A machine that completely destroys items that stands in a room connected to the public hallway with absolutley nothing inbetween but a single row of tables, you are fucking delusional, by your logic we might aswell place the cremator into departures.
> 
> by this response alone you show that you dont play the game, 1 its really easy ro get access to item deleltion, 2 the chef will always kick your skull in if you break into the kitchen ive always seen it happe specially when i try to fry shit 3 getting into the crematarium is just a welder and wrench away lol, or just ask the ai it always opens the door anyway

image
Yeah i don't play the game.

**Comment by Moccha-Bee** [comment]
> > > > > > > Why are we voting on something with objectively better performance and gameplay?
> > > > > > 
> > > > > > 
> > > > > > deep frying things is funny and a core part of chef gameplay no reason it should be removed.
> > > > > 
> > > > > 
> > > > > Are you fucking retarded? This makes it so deepfrying DOESNT COMPLETELY DESTROY THE DEEPFRIED THING IT DOESNT PREVENT DEEPFRIENG
> > > > 
> > > > 
> > > > > making items no longer be eaten and deleted if eaten
> > > > 
> > > > 
> > > > same thing as removing deep frying everyone loves deep frying items to mess with people or get rid of evidence this is some ided type of pr
> > > 
> > > 
> > > A machine that completely destroys items that stands in a room connected to the public hallway with absolutley nothing inbetween but a single row of tables, you are fucking delusional, by your logic we might aswell place the cremator into departures.
> > 
> > 
> > by this response alone you show that you dont play the game, 1 its really easy ro get access to item deleltion, 2 the chef will always kick your skull in if you break into the kitchen ive always seen it happe specially when i try to fry shit 3 getting into the crematarium is just a welder and wrench away lol, or just ask the ai it always opens the door anyway
> 
> image
> Yeah i don't play the game.

i too have 1k hours retard, you still dint adress none of my points #libtard owned

**Comment by katebishop49** [comment]
coders deserve removal from all planes of reality.

**Comment by ghost** [comment]
> coders deserve removal from all planes of reality.

shut up spriter

**Pull request closed without merging**